releases/2.0.1.md

```diff
@@ -0,0 +1,40 @@
+## Arches for Science 2.0.1 Release Notes
+
+
+### Bug Fixes and Enhancements
+
+- Fixes error loading xy-reader [#1601](https://github.com/archesproject/arches-for-science/pull/1601)
+
+### Dependency changes:
+
+```
+Python:
+    Upgraded:
+        None
+JavaScript:
+    Upgraded:
+        Arches 7.6.3 -> 7.6.4
+```
+
+### Breaking Changes
+Due to changes in how projects and apps are configured in the final release of Arches 7.6.3, there are significant project alterations that necessary to run this version of Arches for Science.
+
+### Upgrading Arches for Science 
+1. Start by upgrading your project to run using [Arches 7.6](https://github.com/archesproject/arches/blob/dev/7.6.x/releases/7.6.0.md#upgrading-arches). 
+
+2. Update arches-for-science in your project's package.json file:
+    ```
+    "arches_for_science": "archesproject/arches-for-science#dev/2.0.x"
+    ```
+
+3. Add arches_for_science to your project's dependencies in your pyproject.toml file:
+    ```
+    "arches_for_science>=2.0.0,<2.1.0"
```

Comment:
@chiatt  you might want to change this to 2.0.1 - force the upgrade to 2.0.1.  Otherwise a 2.0.0 version will be fine.